hummingbot/connector/exchange/trading212/trading212_api_user_stream_data_source.py

```
@@ -0,0 +1,60 @@
+import asyncio
+from typing import TYPE_CHECKING, Any, Dict, List, Optional
```

Flake8 (7.2.0)

[error] 'typing.Any' imported but unused

(F401)